### PR TITLE
 Fixed sunburst/sankey issue with rendering

### PR DESCRIPTION
**Pull request by kravets-levko**
Issue getredash/redash#2095

1. Fixed Sankey and Sunburst issue with rendering (did not render on page load).
2. Removed unused (due to removed options) fields from viz editors.
3. Small UI improvements to Cohort viz.

### CODE REVIEW

File: client/app/assets/less/inc/visualizations/sankey.less [hunk 1]
```diff
@@ -12,3 +12,8 @@
   stroke: #000;
   stroke-opacity: .2;
 }
+
+.sankey-visualization-container {
+  height: 300px;
```
Comment (arikfr) [inline client/app/assets/less/inc/visualizations/sankey.less]: Does this make Sankey fixed height?

File: client/app/components/app-header/app-header.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
-<nav class="navbar navbar-default app-header" role="navigation">
-  <div class="container"">
+<nav class="navbar navbar-default navbar-fixed-top app-header" role="navigation">
```
Comment (arikfr) [inline client/app/components/app-header/app-header.html]: Actually we changed it to be not-fixed on purpose. 

I think it's still fixed on shared dashboards -- that's a mistake.

File: client/app/visualizations/sunburst/index.js [hunk 3]
```diff
@@ -40,7 +43,7 @@ export default function init(ngModule) {
 
     const editTemplate = '<sunburst-sequence-editor></sunburst-sequence-editor>';
     const defaultOptions = {
-      height: 300,
```
Comment (arikfr) [inline client/app/visualizations/sunburst/index.js]: If we remove this, we need to remove the option from the viz editor. I'm actually fine with it, if it now occupies the size it is given.

(this is the same for Sankey)

File: client/app/assets/less/inc/visualizations/sankey.less (hunk 1 above)
Comment (kravets-levko) [reply client/app/assets/less/inc/visualizations/sankey.less]: Here (and in other components) - it's the default height (when dashboard is in mobile mode or when auto-height enabled) - just moved from component's config.

File: client/app/components/app-header/app-header.html (hunk 2 above)
Comment (kravets-levko) [reply client/app/components/app-header/app-header.html]: My bad - I had to ask which is right. Will fix now

File: client/app/visualizations/sunburst/index.js (hunk 3 above)
Comment (kravets-levko) [reply client/app/visualizations/sunburst/index.js]: Need to check (probably for other components too). I think, in a case when user can set height of viz directly on dashboard - we don't need such option in editors, it's redundant.

File: client/app/assets/less/inc/visualizations/sankey.less (hunk 1 above)
Comment (arikfr) [reply client/app/assets/less/inc/visualizations/sankey.less]: 👌 this is the same for embeds, right?

File: client/app/visualizations/sunburst/index.js (hunk 3 above)
Comment (arikfr) [reply client/app/visualizations/sunburst/index.js]: I agree. While I can think of uses for this, it's not worth the extra complexity.

File: client/app/assets/less/inc/visualizations/sankey.less (hunk 1 above)
Comment (kravets-levko) [reply client/app/assets/less/inc/visualizations/sankey.less]: Yes, sure

Comment (kravets-levko) [reply client/app/assets/less/inc/visualizations/sankey.less]: And for preview in editors too

**Pull request merged**